cli/src/templates/packages/react-navigation/navigation/drawer-navigator.tsx.ejs

```diff
@@ -25,7 +25,7 @@ export default function DrawerNavigator({ navigation }: Props) {
       />
       <Drawer.Screen
         name="Tabs"
-        component={TabNavigator}
+        component={TabNavigator as React.ComponentType<any>}
```

Comment:
Is there a way to fix the type for the TabNavigator in `'./tab-navigator';` instead?

Reply:
I'll have a deeper look at TabNavigator in ```
'./tab-navigator';
``` the coming days and update the PR accordingly.

Reply:
Yea would like to not have to solve this with `any`